src/main/java/pl/poznan/put/cs/scenariochecker/controller/ScenarioController.java

```diff
@@ -32,4 +33,14 @@ public String count() {
         this.scenarioStrategy.processScenario(this.scenario);
         return String.valueOf(this.scenario.getNumberOfSteps());
     }
-}
\ No newline at end of file
+
+    @GetMapping("/countSpecialSteps")
+        public String countSpecialSteps(){
+            this.scenarioStrategy = new CountSpecialStepsScenarioStrategy();
+            this.scenarioStrategy.processScenario(this.scenario);
+            return String.valueOf(this.scenario.getNumberOfSpecialSteps()-1);
```

Comment:
Maybe instead of subtraction here, you could update stream in `CountSpecialStepsScenarioStrategy` to count true value of special steps?